src/loader.js

```diff
@@ -32,6 +62,9 @@ export function pitch(request) {
     let resultSource;
     if (query.remove) {
       resultSource = '// removed by extract-text-webpack-plugin';
+      if (process.env.NODE_ENV === 'development') {
+        resultSource += hmrCode();
```

Comment:
This would include the HMR Code for every extracted module right? I don't think this should be the case, but also didn't test it yet

Reply:
@sheerun don't rely on `NODE_ENV`, just add `hot` flag :+1:

Reply:
any idea how to output it just one time?